frontend/src/apis/index.js

```diff
@@ -0,0 +1,39 @@
+// Is an abstraction to communicate with backend
+
+class API {
+    BASE_URL = '/api';
+    myFetch = (url, config) =>
```

Comment:
Now, it is not possible to append the headers. If you provide headers, it will just override the current ones. Maybe accept just a few parameters for the fetch function like `makeRequest = (url, method, body, headers) { ... }`.

Reply:
Right, but I feel like if I return to this project after 3 months I will start asking myself why the hell it was not like the original fetch API. Custom API is not always a good idea